platform/persistence/aggregator/src/PersistenceAggregator.js

```diff
@@ -0,0 +1,82 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+/*global define,window*/
+
+define(
+    [],
+    function () {
+        'use strict';
+
+        var METHOD_DEFAULTS = {
+            createObject: false,
+            readObject: undefined,
+            listObjects: [],
+            updateObject: false,
+            deleteObject: false
+        };
+
+        /**
+         * Aggregates multiple persistence providers, such that they can be
+         * utilized as if they were a single object.
```

Comment:
JSDoc should communicate that this does more than just aggregating: it actually routes calls to specific providers.

And as a follow-up: while it can be determined from reading the code, it's not quite clear: what happens if no provider exists for a given space?  Can we add some JSDoc to the constructor to explain?  

note: I'm happy with how DRY `METHOD_DEFAULTS` is-- don't change that!